packages/dev/core/src/Engines/Native/nativePipelineContext.ts

```diff
@@ -278,6 +278,107 @@ export class NativePipelineContext implements IPipelineContext {
         this._engine.setIntArray4(this._uniforms[uniformName]!, array);
     }
 
+    /**
+     * Sets an unsigned integer value on a uniform variable.
+     * @param uniformName Name of the variable.
+     * @param value Value to be set.
+     */
+    public setUInt(uniformName: string, value: number): void {
+        const cache = this._valueCache[uniformName];
+        if (cache !== undefined && cache === value) {
+            return;
+        }
+
+        if (this._engine.setUInt(this._uniforms[uniformName]!, value)) {

```

Comment:
@bghgary, @CedricGuillemet, I guess we d need to support those in native now :-)